### PR TITLE
add comment viewing on each article and update homepage article to sh…

### PR DESCRIPTION
**Pull request by JoeAshcroft**
…ow a random article each time

### CODE REVIEW

File: src/Components/CommentCard.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const CommentCard = ({ author, body, created_at, votes }) => {
+  const dateFormatter = (created_at) => {
+    return new Date(created_at).toLocaleDateString();
```
Comment (HeatherEver) [inline src/Components/CommentCard.jsx:3]: nice ⭐

File: src/Components/Nav.jsx [hunk 2]
```diff
@@ -13,9 +13,9 @@ const Nav = () => {
         <Link to="/topics">
           <li className="nav-li">Topics</li>
         </Link>
-        <Link to="/comments">
+        {/* <Link to="/comments">
           <li className="nav-li">Comments</li>
-        </Link>
+        </Link> */}
```
Comment (HeatherEver) [inline src/Components/Nav.jsx]: if this isn't being used, please remove :)

**Review by HeatherEver — COMMENTED** [review]
Nice work so far put there are a couple of things to note: 
- you're a useEffect in your Comments component to get a single article, this is kind of pointless as you should just render the Comments component in your SingleArticle component, then Comments only job is to fetch the comments and display them. This is also causing the comments to appear before the article they are on when the button is clicked, so I would really consider separating these :) 
- if an article doesn't have any comments it currently just looks like your button isn't working correctly, it would be nice to display a message to the user to say there are no comments :)

**Comment by robert112233** [comment]
Great work Joe, Article 37 has no comments and your message works perfectly. Your layout makes alot of sense now. Merge!

**Pull request merged**